pkg/controllers/state/suite_test.go

```diff
@@ -64,13 +64,15 @@ var _ = AfterSuite(func() {
 })
 
 var _ = BeforeEach(func() {
+	// fmt.Printf("Running setup...\n")
```

Comment:
Remove commented code. Looks like you're not changing anything in this file. Let's remove this file from the PR.